plugins/modules/source_control/gitlab/gitlab_group.py

```diff
@@ -61,6 +61,22 @@
     choices: ["private", "internal", "public"]
     default: private
     type: str
+  project_creation_level:
+    description:
+      - Determine if developers can create projects in the group
+    choices: ["developer", "maintainer", "noone"]
+    default: developer
+    type: str
```

Comment:
```suggestion
    type: str
    version_added: 3.5.0
```
(same for the others)